src/docs/index.md

```diff
@@ -128,6 +128,96 @@ title: Akka.NET Documentation
+	- [Thread safety of custom processing stages](streams/customstreamprocessing#thread-safety-of-custom-processing-stages)
+	- [Extended Flow Combinators with Custom Operators](streams/customstreamprocessing#extending-flow-combinators-with-custom-operators)
+- [Integration](streams/integration)
+	- [Integrating with Actors](streams/integration#integrating-with-actors)
+	- [Integrating with External Services](streams/integration#integrating-with-external-services)
+	- [Integrating with Reactive Streams](streams/integration#integrating-with-reactive-streams)
+- [Error Handling](streams/error-handling)	
+	- [Supervision Strategies](streams/error-handling#supervision-strategies)
+	- [Errors from SelectAsync](streams/error-handling#errors-from-selectasync)
+- [Working with streaming IO](streams/workingwithstreamingio)
+	- [Streaming TCP](streams/workingwithstreamingio#streaming-tcp)
+	- [Streaming File IO](streams/workingwithstreamingio#streaming-file-io) 
+- [Pipelining and Parallelism](streams/pipeliningandparallelism)
+	- [Pipelining](streams/pipeliningandparallelism#pipelining)
+	- [Parallel processing](streams/pipeliningandparallelism#parallel-processing)
+	- [Combining pipelining and parallel processing](streams/pipeliningandparallelism#combining-pipelining-and-parallel-processing)
+- [Testing streams](streams/testingstreams)
+	- [Built in sources, sinks and combinators](streams/testingstreams#built-in-sources-sinks-and-combinators)
+	- [TestKit](streams/testingstreams#testkit)
+	- [Streams TestKit](streams/testingstreams#streams-testkit)
+	- [Fuzzing Mode](streams/testingstreams#fuzzing-mode)
+- [Overview of built-in stages and their semantics](streams/builtinstages)
+	- [Source stages](streams/builtinstages#source-stages)
+	- [Sink stages](streams/builtinstages#sink-stages)
+	- [Additional Sink and Source converters](streams/builtinstages#additional-sink-and-source-converters)
+	- [File IO Sinks and Sources](streams/builtinstages#file-io-sinks-and-sources)
+	- [Flow stages](streams/builtinstages#flow-stages)
+	- [Simple processing stages](streams/builtinstages#simple-processing-stages)
+	- [Asynchronous processing stages](streams/builtinstages#asynchronous-processing-stages)
+	- [Timer driven stages](streams/builtinstages#timer-driven-stages)
+	- [Backpressure aware stages](streams/builtinstages#backpressure-aware-stages)
+	- [Nesting and flattening stages](streams/builtinstages#nesting-and-flattening-stages)
+	- [Fan-in stages](streams/builtinstages#fan-in-stages)
+	- [Fan-out stages](streams/builtinstages#fan-out-stages)
+	- [Watching status stages](streams/builtinstages#watching-status-stages)
+- [Streams Cookbook](streams/cookbook)
+	- [Introduction](streams/cookbook#introduction)
+	- [Working with Flows](streams/cookbook#working-with-flows)
+	- [Working with Graphs](streams/cookbook#working-with-graphs)
+	- [Working with Rate](streams/cookbook#working-with-rate)
+	- [Working with IO](streams/cookbook#working-with-io)
```

Comment:
This could probably also be skipped, it handles cases for ByteString's that you usually only need when you work with tcp.